### ISSUE TITLE
[Bug] Record values no longer available in  contentType Edit/create 

### ISSUE THREAD

**Issue by cdowdy** [bolt/bolt#6768]
Description:  
-------------  
in a contentType edit/create screen with ``allowtwig: true`` set in an HTML body field, ex:  

```yaml  
entries:
    name: Entries
    singular_name: Entry
    fields:  
       # other fields
        slug:
            type: slug
            uses: title
        body:
            type: html
            height: 300px
            allowtwig: true
```

``{{ record.slug }}`` or any other ``{{record.{value_here }}`` are no longer available to extensions with twig functions

Details
-------

 - Relevant Bolt Version: [ noticed 3.2.13 ]
 - Install type: [ Zip/tarball install | Composer install ]
 - PHP version: [ 7.0 ]
 - Used web server: [ Apache Apache/2.4.18  | Nginx 1.10.2] ]

Reproduction
------------

​
 1. **Bug summary**:   
    ``{{ record.values }}``   cannot be "fetched" 
 2. **Specifics**:
    * first noticed bolt 3.2.13 & bolt 3.2.14
    * installed through composer  
    * screenshots:  
       * in the backend editor:  
![record-test-contenttype](https://user-images.githubusercontent.com/5969974/27345784-dc79dd2a-55b8-11e7-9fda-d3e78dbb966f.png)  
        * result:  
        
![record-test-contenttype-result](https://user-images.githubusercontent.com/5969974/27345813-f2647abe-55b8-11e7-8177-675a89da9258.png)

 3. **Steps to reproduce**:
    * composer install bolt
    * fill with lorem ipsum records  
    * allowtwig true for the "body" field  
    * edit record and try to get and field defined in the contentType (slug, image etc)
 4. **Expected result**: 
    * the slug or image would be given
    * any value with ``{{ record.value }}`` would be rendered/printed in the body field
 5. **Actual result**: 
    * record values are removed  

If you use any other twig function, ie: ``{{ "now"|date("m/d/Y") }}`` they are output:  

![other-twig-contenttype](https://user-images.githubusercontent.com/5969974/27345999-9e574de2-55b9-11e7-8620-5fe3b3673301.png)

result:  
![other-twig-result](https://user-images.githubusercontent.com/5969974/27346003-a435a06a-55b9-11e7-8416-29c5109b1229.png)

**Comment by rossriley** [comment]
OK, I've had a look at this one and I think there must be something else affecting this.

I've followed the reproduction steps on both 3.2.13 and 3.2.14 and I think I get expected behaviour.

That is, 1: Add allowtwig capability:

```
        body:
            type: html
            height: 300px
            allowtwig: true
```

Then I added this to the body HTML:

```html
<p>{{ record.id }}</p>

<p>{{ record.title }}</p>

<p>{{ record.link }}</p>

<p>{{ record.datepublished|date('d/m/Y') }}</p>

<p>{{ record.slug }}</p>
```

Hit save and visit the frontend. Using the above config I get:

-----

1

Test Number One

/page/test-number-one

03/07/2017

test-number-one

---

If I disable the allowtwig setting and reload I see the raw Twig code:

---

{{ record.id }}

{{ record.title }}

{{ record.link }}

{{ record.datepublished|date('d/m/Y') }}

{{ record.slug }}

---

One thing that I can guess at, is there was some previous problems with rendering markdown inside the `fields()` call, in your frontend template are you rendering the fields yourself via `{{ record.body }}` or is this being done by the `fields` method?

**Comment by cdowdy** [comment]
well I'll be damned. I just did a random composer update (still on 3.2.14) and created a new contentType and it worked just as you described: 

setup:  
![bolt-ctt-setup](https://user-images.githubusercontent.com/5969974/27793396-d85b33ac-5fcb-11e7-846d-8e01c5a12965.png)
  
contentType editor:  
![bolt-ct-body](https://user-images.githubusercontent.com/5969974/27793410-e53fc236-5fcb-11e7-9750-c726ce096a86.png)
   

Then just using ``{{ record.body }}`` only 
result:  
![bolt-result](https://user-images.githubusercontent.com/5969974/27793414-ede474f4-5fcb-11e7-8b46-00998a5ea2ef.png)
  

and I wish I could tell you what was different between these two but I already burned my test environment I used for the original bug test case haha which suuuuuuuuucks (i'll keep this one around longer).  

Thanks for double checking @rossriley sorry for the extra work I caused!
